external/source/exploits/drunkpotato/RogueWinRM exe/main.c

```diff
@@ -0,0 +1,49 @@
+/*

```

Comment:
Is a stand alone executable with this entry point required? It looks like the bulk of the exploit is implemented as an RDLL and if that's the case then I'm not sure this is necessary.

Reply:
Hello !

I created a standalone executable for 2 reasons:
1 : Every time I want to reverse engineer a dll exploit, I regret there is no standalone executable to try and tweak the exploit. So I created this executable as a convenient way to try it.
2 : I think it is easier to debug in visual studio.